lib/node_modules/@stdlib/string/next-grapheme-cluster-break/docs/types/index.d.ts

```diff
@@ -0,0 +1,43 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2020 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+// TypeScript Version: 2.0
+
+/**
+* Returns the next extended grapheme cluster break in a string after a specified position.
+*
+* @param str - input string
+* @param idx - position
+* @throws first argument must be a string
+* @throws second argument must be a nonnegative integer
```

Comment:
I think we can allow a negative integer for the `pos` as a shorthand for `str.length-<n>`. This is common for other APIs which perform searches over an input value (string, array, or otherwise).

Reply:
Makes sense, will do this.